### PR TITLE
Integration of title and section

### PR DESCRIPTION
**Pull request by Nhacsam**

### CODE REVIEW

File: wp-content/themes/nuitdeboo-child/templates/module-home-social.php [hunk 1]
```diff
@@ -1,12 +1,5 @@
-<div id="socialhome">
-	<div class="container padded">
-		<div class="row">
-			<div class="col-md-8 col-md-offset-2">
-				<h2>R&eacute;seaux sociaux</h2>
-			</div>
-		</div>
-	</div>
-
+<section id="socialhome" class="section section--gray">
```
Comment (alexsegura) [inline wp-content/themes/nuitdeboo-child/templates/module-home-social.php:1]: Missing `</section>` tag at the end of the file

**Comment by alexsegura** [comment]
On small screens, section text is cut:

capture d ecran 2016-04-23 a 19 04 14

**Comment by alexsegura** [comment]
Fixed and merged in 435218a27c79353342d60fde4a935e8fdd888f86

File: wp-content/themes/nuitdeboo-child/templates/module-home-social.php (hunk 1 above)
Comment (Nhacsam) [inline wp-content/themes/nuitdeboo-child/templates/module-home-social.php:1]: Bien vu

**Comment by Nhacsam** [comment]
Merci :)

**Pull request closed without merging**